packages/sdk/src/services/kwentaToken.ts

```diff
@@ -344,20 +458,46 @@ export default class KwentaTokenService {
 		return this.sdk.transactions.createContractTxn(RewardEscrow, 'vest', [ids])
 	}
 
+	public vestTokenV2(ids: number[]) {
+		const { RewardEscrowV2 } = this.sdk.context.contracts
+
+		if (!RewardEscrowV2) {
+			throw new Error(sdkErrors.UNSUPPORTED_NETWORK)
+		}
+
+		return this.sdk.transactions.createContractTxn(RewardEscrowV2, 'vest', [ids])
+	}
+
 	public stakeKwenta(amount: string | BigNumber) {
 		return this.performStakeAction('stake', amount)
 	}
 
 	public unstakeKwenta(amount: string | BigNumber) {
-		return this.performStakeAction('unstake', amount)
+		return this.performStakeAction('unstake', amount, { escrow: false, version: 1 })
 	}
 
 	public stakeEscrowedKwenta(amount: string | BigNumber) {
```

Comment:
should this be renamed to `stakeEscrowedKwentaV2`?

Reply:
`stakeEscrowedKwentaV2` has already been defined in the following lines of the code. However, I went through the code and found that I had passed the wrong version into this function as I figured out why you ask this question. Thanks for the reminder.